src/CGB_Infrared.md

```diff
@@ -0,0 +1,49 @@
+# CGB Infrared
+
+The GBC comes with an infrared port on the very top of the handheld,
+which can be used to send and receive IR signals via its two separate
+diodes. This functionality is not limited to other GBCs - it can be
+used to communicate with external devices as well.
+
+Note that the GBA does not include an infrared port and, as such, this
+method of communication is unavailable in its CGB compatibility mode.
+
+## FF56 - RP - Infrared Communications Port
+
+This register allows data to be transmitted and received through the
+infrared port.
+
+Bit | Name             | Usage notes                     | R/W?
+----|------------------|---------------------------------|-------------
+6-7 | Data read enable | 0=Disable, 3=Enable             | Read/Write
+ 1  | Read data        | 0=Receiving IR Signal, 1=Normal | Read Only
+ 0  | Write data       | 0=LED Off, 1=Led On             | Read/Write
+
+In order to read data, both Bit 6 and 7 must be set. Note that if Bit 0
+is not cleared, the CGB will end up reading its own IR signal.
```

Comment:
```suggestion
is also set, the CGB will end up reading its own IR signal.
```

Reply:
The wording here ("not cleared") implies that, for proper usage, you _should_ clear the bit. This is correct, so maybe the stronger wording is helpful.

Reply:
It's possible to write once to enable reading, and once again to disable the emitter, for example. So maybe "while bit 0 is also set, ..."?